OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -242,6 +239,7 @@
     <Compile Include="Widgets\WorldInteractionControllerWidget.cs" />
     <Compile Include="Graphics\PaletteReference.cs" />
     <Compile Include="Graphics\TerrainSpriteLayer.cs" />
+    <Compile Include="FileSystem\InstallShieldCABExtractor.cs" />
```

Comment:
Please revert everything in this file except for this line. Some of these changes are even preventing me from running the game.